data/communities.csv

```diff
@@ -62,4 +62,5 @@ name,description,website,region,category,country,city_state,member_count,status,
 "OpenChain China Work Group","OpenChain Project","https://openchainproject.org/participate","Eastern Asia","Others","China",,,"Active","OpenChain Chapter","hybrid"
 "Ubuntu Korea Community","Ubuntu Chapter","https://ubuntu-kr.org","Eastern Asia","Others","South Korea",,,"Active","Ubuntu Chapter","hybrid"
 "Kaiyuanshe Alliance City Communities","Kaiyuanshe","https://kaiyuanshe.cn/community","Eastern Asia","Others","China",,,"Active","Kaiyuanshe City Chapters","hybrid"
+"GoFr Community","Open-source Golang framework for accelerated microservice development","https://gofr.dev/","Southern Asia","Cloud Native","India,Bangalore","11000","Active","CNCF Ecosystem","hybrid"
```

Comment:
Unless this is a chapter of the CNCF, you don't need to mention it. Additionally, please follow the convention followed in the CSV.
```suggestion
"GoFr Community","Open-source Golang framework for accelerated microservice development","https://gofr.dev/events","Southern Asia","Cloud Native","Bangalore, India","11000","Active",","hybrid"
```